front-end/src/renderer/components/RecoveryPhrase/Import.vue

```diff
@@ -100,6 +108,13 @@ watch(
     }
   },
 );
+
+watchEffect(() => {
```

Comment:
shouldClear needs to be watched, but why inputRefs? This causes manual altering of any one of the words to clear all the words, I think.